bindings/python/examples/how_tos/outputs/features.py

```diff
@@ -73,4 +73,4 @@
 )
 outputs.append(nft_output)
 
-print(json.dumps(outputs, indent=2))
+print(json.dumps([asdict(o) for o in outputs], indent=2))
```

Comment:
Doesn't this need the `from dataclasses import asdict` import?

Reply:
it's weird It was working without it :D.  But I've added it just to make sure